src/tokens/corporate/globals/colors.json

```diff
@@ -3,6 +3,12 @@
     "type": "color",
     "brand-primary": {
       "value": "#003c71"
+    },
+    "bsc-logo-primary": {
+      "value": "#00467F"
+    },
+    "bsc-logo-secondary": {
+      "value": "#6A737B"
```

Comment:
can we lowercase the hex values here. let's also check the linters are working as expected. i thought the autofix was set up to update those.